examples/pytorch/elastic/imagenet/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=pytorch/pytorch:1.13.1-cuda11.6-cudnn8-runtime
+ARG BASE_IMAGE=nvcr.io/nvidia/pytorch:24.01-py3
```

Comment:
Can we add a similar comment as https://github.com/kubeflow/katib/blob/250e9d176f047f524d37bb417db46b2563d0a72d/examples/v1beta1/trial-images/pytorch-mnist/Dockerfile.gpu#L1-L3?